monitoring/snippets/v3/uptime-check-client/snippets_test.py

```diff
@@ -59,13 +58,13 @@ def uptime():
         yield uptime
 
 
-def test_create_and_delete(capsys):
+def test_create_and_delete(capsys) -> None:
```

Comment:
`capsys` most probably is no longer needed

Reply:
done